src/liblibc/lib.rs

```diff
@@ -4827,7 +4827,6 @@ pub mod funcs {
                                    mibp: *mut c_int,
                                    sizep: *mut size_t)
                                    -> c_int;
-            pub fn getdtablesize() -> c_int;
```

Comment:
I checked if it is used in somewhere else, and no one was using that.
But anyway I re-added that line,
and made it only invisible from aarch64-android.